frontend/src/metabase/query_builder/components/SavedQuestionIntroModal.tsx

```diff
@@ -48,18 +44,19 @@ export const SavedQuestionIntroModal = ({
   const { title, message } = getLabels(question);
 
   return (
-    <Modal isOpen={isShowingNewbModal}>
-      <ModalContent title={title} className={cx(CS.textCentered, CS.py2)}>
-        <div className={cx(CS.px2, CS.pb2, CS.textParagraph)}>{message}</div>
-        <div className={cx("Form-actions", CS.flex, CS.justifyCenter, CS.py1)}>
-          <button
-            className={cx(ButtonsS.Button, ButtonsS.ButtonPrimary)}
-            onClick={onClose}
-          >
-            {t`Okay`}
-          </button>
-        </div>
-      </ModalContent>
-    </Modal>
+    <Modal.Root opened={isShowingNewbModal} onClose={onClose} size={500}>
+      <Modal.Overlay />
+      <Modal.Content p="md">
+        <Modal.Header mb="md">
+          <Modal.Title>{title}</Modal.Title>
+        </Modal.Header>
+        <Modal.Body ta="center">
+          <Box mb="md" ta="left">
```

Comment:
You can convert it to `Text` and play with text settings to make it look line in the designs.

Reply:
https://github.com/metabase/metabase/pull/45148/commits/f2884d883daee4c531a2a2e4e96642a2e4615aee